Frends.AzureBlobStorage.ListContainers/Frends.AzureBlobStorage.ListContainers/Definitions/ContainerInfo.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+
+namespace Frends.AzureBlobStorage.ListContainers.Definitions;
+
+/// <summary>
+/// Represents information about an Azure Blob Storage container.
+/// </summary>
+public class ContainerInfo
+{
+    /// <summary>
+    /// Container name.
+    /// </summary>
+    /// <example>"mycontainer01"</example>
```

Comment:
Please do not quote those examples. Otherwise e.g. in Connection class you also have strings and there are no quotes around those examples.